### PR TITLE
Add TSE flag for liveboard breakpoint

### PR DESCRIPTION
**Pull request by goutham-sidhardha**

### CODE REVIEW

File: src/embed/ts-embed.ts [hunk 1]
```diff
@@ -492,7 +493,9 @@ export class TsEmbed {
         if (spriteUrl) {
             queryParams[Param.IconSpriteUrl] = spriteUrl.replace('https://', '');
         }
-
+        if (enable2ColumnLayout !== undefined) {
```
Comment (AdityaMittal31) [inline src/embed/ts-embed.ts:496]: This is specific to LB right? Please move it in liveboardEmbed view config.

[the remark this answers is not in the document]
Comment (goutham-sidhardha) [reply src/embed/ts-embed.ts:496]: we are also able to access liveboard from the FullApp option right

[the remark this answers is not in the document]
Comment (goutham-sidhardha) [reply src/embed/ts-embed.ts]: updated

File: src/embed/liveboard.ts [hunk 2]
```diff
@@ -352,6 +369,9 @@ export class LiveboardEmbed extends V1Embed {
         if (liveboardV2 !== undefined) {
             params[Param.LiveboardV2Enabled] = liveboardV2;
         }
+        if (enable2ColumnLayout !== undefined) {
+            params[Param.Enable2ColumnLayout] = enable2ColumnLayout;
```
Comment (adityamittal3107) [inline src/embed/liveboard.ts:373]: Add UT

File: src/embed/liveboard.ts [hunk 3]
```diff
@@ -269,6 +269,22 @@ export interface LiveboardViewConfig
      * ```
      */
     enableAskSage?: boolean;
+    /**
+     * This flag is used to enable the 2 column layout in liveboard
+     *
+     * @type {boolean}
+     * @default false
+     * @version SDK: 1.29.0 | ThoughtSpot:10.1.0.cl
```
Comment (adityamittal3107) [inline src/embed/liveboard.ts:277]: 1.29 version is for 9.12. For 10.1 It will be 1.32

File: src/embed/app.ts [hunk 4]
```diff
@@ -359,6 +359,22 @@ export interface AppViewConfig extends Omit<ViewConfig, 'visibleTabs'> {
      * @default false
      */
     collapseSearchBarInitially?: boolean;
+    /**
+     * This flag is used to enable the 2 column layout in liveboard
+     *
+     * @type {boolean}
+     * @default false
+     * @version SDK: 1.29.0 | ThoughtSpot:10.1.0.cl
```
Comment (adityamittal3107) [inline src/embed/app.ts:367]: same as below

**Review by adityamittal3107 — APPROVED** [review]

**Pull request merged**